x/leverage/types/flag.go

```diff
@@ -0,0 +1,6 @@
+package types
+
+const FlagEnableLiquidatorQuery = "enable-liquidator-query"
+
+// EnableLiquidator must be set to true to enable QueryLiquidationTargets
+var EnableLiquidator *bool
```

Comment:
ouch. A global, public pointer is a something we should definitely avoid.
Why not adding this flag to app cmd, and passing it through `AppModule` down to the keeper?
Maybe here: https://github.com/umee-network/umee/blob/main/cmd/umeed/cmd/root.go#L188

Reply:
Is there a way to get from `leverage.AddModuleInitFlags(startCmd)` to the keeper constructor? If so, I can place the flag at `keeper.EnableLiquidatorQuery`

Reply:
Through `appOpts`. Check here: https://github.com/umee-network/umee/blob/main/cmd/umeed/cmd/app_creator.go#L53

Reply:
You have also access to it in `New` in app.go.